documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_1_Security_Perspective/4_1_3_Securing_the_Platform.md

```diff
@@ -0,0 +1,59 @@
+# Securing the Platform
+
+Security of data processing in an IDS component depends on the system security of the utilized component. The hardware and software components that are critical to ensure the confidentiality and integrity of the transmitted and processed data form the Trusted Computing Base (TCB). Different deployment scenarios and their respective TCBs are described in the [first subsection](#deployment-scenarios). The [following subsection](#hardware-security-features) provides some background on hardware security features that might be used for securing the platform. Finally, the [last subsection](#platform-security-requirements) provides an overview of essential security requirements and how to achieve them.
+
+## Deployment Scenarios
+In general, there are three different possibilities for the combinations of platform and connector service instances:
+* A 1:1 mapping with a connector being a single device consisting of one platform instance and one connector service instance
+* A 1:n mapping with multiple connectors on one physical device, i.e., a system with one platform instance and multiple connector service instances
+* A n:m mapping with multiple devices offering a distributed deployment for multiple connector service instances (e.g. with Kubernetes)
+
+The figure below shows the three options with their respective trusted computing bases:
+![Deployment Scenarios](./media/deployment_scenarios.png)
+#### _Fig. 4.1.3.1: Deployment Scenarios_
+
+The TCB consists of the following components:
+* All **hardware** components with access to unencrypted app data.
+* **Firmware / bootloader / UEFI** representing all software components used to bootstrap the system and initialize the hardware before starting the kernel.
+* Optionally a **hypervisor** which may be used to better isolate multiple connectors on a device by providing a Virtual Machine (VM) for each of them. The impact on the TCB is illustrated for the 1:n mapping in a comparison to a solution with OS-level virtualization, i.e., containers.
```

Comment:
Don't know how much we can trust a hypervisor in regards to isolation more than a container runtime.

Reply:
```suggestion
* Optionally a **hypervisor** which may be used to isolate multiple connectors on a device by providing a Virtual Machine (VM) for each of them. The impact on the TCB is illustrated for the 1:n mapping in a comparison to a solution with OS-level virtualization, i.e., containers.
```